js/researches/calculateFleschReading.js

```diff
@@ -33,8 +37,17 @@ module.exports = function( paper ) {
 	}
 
 	var numberOfSyllables = countSyllables( text, locale );
+	switch( language ) {
+		case "nl":
+			var syllablesPer100Words = numberOfSyllables * ( 100 / numberOfWords );
+			score = 206.84 - ( 0.77 * syllablesPer100Words ) - ( 0.93 * ( numberOfWords / numberOfSentences ) );
```

Comment:
Considering the `numberOfWords / numberOfSentences` calculation is being done multiple times, it might be a good idea to create a `getAverageWordsPerSentence` function.